.github/workflows/distcheck.yaml

```diff
@@ -217,140 +221,30 @@ jobs:
           then
             ./configure --with-mpi=/opt/amazon/openmpi \
                         --with-libfabric=/opt/amazon/efa \
-                        --with-cuda=/usr/local/cuda/ \
                         --enable-tests \
-                        --enable-platform-aws
+                        --enable-platform-aws \
+                        --with-cuda=/usr/local/cuda/
           else
-            ./configure --with-libfabric=/opt/amazon/efa \
-                        --enable-neuron \
-                        --enable-platform-aws
-          fi
-          make -j "$(nproc)"
-
-      - name: Run Dist Check
-        run: make distcheck
-
-      - name: Upload build logs
-        if: failure()
-        uses: actions/upload-artifact@v4
-        with:
-          name: ${{ matrix.cc }}-${{ matrix.cc-variant }}-${{ matrix.sdk }}-config.log
-          path: config.log
-          if-no-files-found: ignore
-
-  unit-tests:
-    runs-on: ubuntu-22.04
-    strategy:
-      matrix:
-        cc-variant:
-          - latest
-          - legacy
-        cc:
-          - gcc
-          - clang
-        sdk:
-          - cuda
-          - neuron
-        include:
-          - cc-variant: latest
-            cc: clang
-            cc-version: 18
-          - cc-variant: latest
-            cc: gcc
-            cc-version: 13
-
-    name: u2204/${{ matrix.sdk }}/${{matrix.cc}}(${{matrix.cc-variant}})/unit-tests/
-    steps:
-      - uses: actions/checkout@v4
-      - name: Configure Neuron SDK Repository
```

Comment:
Why don't you need this "Configure Neuron SDK Repository" step anymore?

Reply:
Never needed this